docs/docsite/rst/intro_windows.rst

```diff
@@ -94,8 +94,12 @@ You can specify which authentication option you wish to use by setting it to the
 Certificate
 +++++++++++
 
-Certificate authentication is similar to SSH where a certificate is assigned to a local user and instead of using a password to authenticate a certificate is used instead.
+Certificate authentication is similar to SSH because it uses both a public and private key. This removes the need for using a password.
```

Comment:
Don't want to be pedantic but I think keeping the local user bit is important. You can use certificate auth for domain accounts and this change might give the wrong impression.